az-core/src/main/java/azkaban/Constants.java

```diff
@@ -883,7 +883,7 @@ public static class FlowParameters {
     public static final String FLOW_PARAM_ALLOW_RESTART_ON_STATUS = "flow.retry.statuses";
 
     // Constant to define at most how many times can restart the flow
-    public static final String FLOW_PARAM_RESTART_COUNT = "flow.max.retries";
+    public static final String FLOW_PARAM_MAX_RETRIES = "flow.max.retries";
```

Comment:
this config is introduced for the flow-resumability project and is of no use before this PR.